lib/providers/googlemaps.js

```diff
@@ -142,4 +143,74 @@ var GoogleMaps = Provider.extend(function (self, opts) {
     };
 });
 
+GoogleMaps.fixture = function(opts) {
+    /**function:GoogleMaps.fixture(opts)
+
+    Returns an HTTP resource fixture for a GoogleMaps location query.
+
+    :param string opts.query:
+        The address that is to be queried. Required.
+
+    :param array_of_strings opts.address_list:
+         An array of the list of `formatted_address`'s that should be sent in
+         the response. If response_data is included, this will be ignored.
+         Defaults to ``[]``.
+
+    :param string opts.request_url:
+        URL for the HTTP request. Defaults to
+        ``"http://maps.googleapis.com/maps/api/geocode/json"``.
+
+    :param array_of_objects opts.response_data:
+        An array of objects that represents the response from the gmaps API.
+
+    Usage:
+
+    .. code-block:: javascript
+
+        tester
+            .setup(function(api) {
+                api.http.fixtures.add(
+                    GoogleMaps.fixture({
+                        request: "New Street",
+                        address_list: [
+                            "New Street 1", "New Street 2",
+                        ],
+                    });
+                );
+            });
+
+    */
+    opts = _.defaults(opts || {}, {
+        request_url: "http://maps.googleapis.com/maps/api/geocode/json",
+        address_list: [],
+    });
+
+    if (typeof opts.query === 'undefined') {
+        throw new FixtureParameterMissingError(
+            "'query' option is required when creating a GoogleMaps fixture."
+        );
+    }
+
+    if (!opts.response_data) {
+        opts.response_data = {
+            status: "OK",
+            results: _.map(opts.address_list, function(address) {
+                return {formatted_address: address};
+            }),
+        };
+    }
+
+    return {
+        request: {
+            method: "GET",
+            url: opts.request_url,
+            params: {'address': opts.query},
+        },
+        response: {
+            code: "200",
+            data: opts.response_data,
+        },
+    };
+};
```

Comment:
It seems a bit weird having the fixture code be part of the implementation's code. Not something for this PR, but wondering if we want the fixture part in a separate module?